### PR TITLE
Added Comments and Updated README.md

### PR DESCRIPTION
**Pull request by huahaoxie0921**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,2 @@
 # MCNPC
 The first function is distractor.test.
-Test
```
Comment (YuJojo) [inline README.md:3]: Should not delete this line.

File: distrator_test.R [hunk 2]
```diff
@@ -3,7 +3,7 @@ distractor.check=function(Q){
   Q=as.matrix(Q)
   J=length(unique(Q[,1]))
   K=ncol(Q)-2
-  
+  # [Test] Add Comments
```
Comment (YuJojo) [inline distrator_test.R:6]: Unnecessary Comment.

File: README.md (hunk 1 above)
Comment (huahaoxie0921) [reply README.md:3]: Ack

**Review by YuJojo — APPROVED** [review]
LGTM

**Pull request merged**